app/services/square_manager/payment_creator.rb

```diff
@@ -32,5 +32,14 @@ def create_payment_body
         location_id: location_id
       }
     end
+
+    def which_access_token_from_location_id(location_id:)
```

Comment:
nit `access_token_from_location_id`

Reply:
is this codepath tested any way?

Reply:
sure changed the name, and I don't think it's actually being hit in unit tests, let me see if I can whip one up

Reply:
Just added a unit test for this which I'm happy about 👌